WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -256,4 +301,14 @@ private Person getCurrentPerson() {
 
         return personDetailFragment.loadPerson();
     }
+
+    @Override
+    public void onFetchMorePeople() {
+        if (mPeopleEndOfListReached) {
+            return;
+        }
+        Blog blog = WordPress.getCurrentBlog();
```

Comment:
I really should just use Github instead of going through the emails.. I came here to say the same thing just to see that you've already edited it 👍